tests/mgbench/benchmark_context.py

```diff
@@ -0,0 +1,56 @@
+# Describes all the information of single benchmark.py run.
+class BenchmarkContext:
```

Comment:
For this, you can use https://docs.python.org/3/library/dataclasses.html, but I think for that, we'll force users to use Python 3.7 (double check the version)

Reply:
Yes I am aware of this, but we need some custom loggic in constructor here + it may even grow in future, and it is also more visible what is happening. But the downside is manually adding new flags.